packages/inula-adapter/examples/$children/FruitList.jsx

```
@@ -0,0 +1,28 @@
+      useInstance()
+        .$children.map(({ fruitName, isChecked }) => `${fruitName}:${isChecked ? '✓' : 'X'}`)
+        .join(', ') //returns string like "apple:✓, pear:X, motorcycle:✓"
+    );
+  }
+
+  return (
+    <div>
+      <ul>
+        {fruits.map(fruit => (
+          <FruitItem fruitName={fruit} />
+        ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)